redash/query_runner/snowflake.py

```diff
@@ -123,10 +123,18 @@ def _run_query_without_warehouse(self, query):
             cursor.close()
             connection.close()
 
-        return data, error
+        return data, error    
+    
+    def _database_name_with_schema(self):
```

Comment:
The name of this function sounds like it returns the database name with schema. How about

```suggestion
    def _database_name_includes_schema(self):
```

Reply:
👍